gamestonk_terminal/cryptocurrency/crypto_controller.py

```diff
@@ -7,36 +8,62 @@
 from gamestonk_terminal import feature_flags as gtff
 from gamestonk_terminal.helper_funcs import get_flair
 from gamestonk_terminal.menu import session
-from gamestonk_terminal.cryptocurrency import pycoingecko_view
-from gamestonk_terminal.cryptocurrency import coinmarketcap_view as cmc_view
+from gamestonk_terminal.cryptocurrency import (
+    binance_model,
+    pycoingecko_view,
+    coinmarketcap_view as cmc_view,
+)
 
 
 class CryptoController:
 
-    CHOICES = ["help", "q", "quit", "load", "view", "top"]
+    CHOICES = [
+        "help",
+        "q",
+        "quit",
+        "load",
+        "view",
+        "top",
+        "trend",
+        "book",
+        "trades",
+        "candle",
+        "balance",
+        "add",
+    ]
 
     def __init__(self):
         """CONSTRUCTOR"""
 
         self.crypto_parser = argparse.ArgumentParser(add_help=False, prog="crypto")
         self.crypto_parser.add_argument("cmd", choices=self.CHOICES)
         self.current_coin = None
+        self.current_currency = None
         self.current_df = pd.DataFrame()
 
     @staticmethod
-    def print_help(current_coin):
+    def print_help(cont):
         """Print help"""
         print("\nCryptocurrency:")
         print("   help          show this menu again")
         print("   q             quit this menu, and shows back to main menu")
         print("   quit          quit to abandon program")
-        print(f"\nCurrent Coin: {current_coin}")
+        print(f"\nCurrent Coin: {cont.current_coin}")
         print("")
+        print("Coingecko:")
         print("   load          load cryptocurrency data")
         print("   view          load and view cryptocurrency data")
+        print("   trend         show top 7 trending coins")
         print("")
+        print("CoinMarketCap:")
         print("   top           view top coins from coinmarketcap")
         print("")
+        print("Binance:")
+        print("   add           add coin as current")
```

Comment:
One thing is that this description "add coin as current" needs to be the same in FEATURES markdown, and also in table of contents of crypo README.

Also, this command should be "select" or so, and description can be improved